integration/sawtooth_integration/tests/test_namespace_permission.py

```diff
@@ -55,6 +60,16 @@ def get_state(address):
     return base64.b64decode(response['data'])
 
 
+def get_state_prefix(prefix):
```

Comment:
consider clearer method name: get_state_by_prefix

Reply:
I made the change. Thanks!